src/main/java/com/epam/test_generator/dto/RegistrationUserDTO.java

```diff
@@ -0,0 +1,97 @@
+package com.epam.test_generator.dto;
+
+import org.hibernate.validator.constraints.Email;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class RegistrationUserDTO {
+
+    @NotNull
+    @Size(min = 1, max = 255)
+    private String name;
+
+    @NotNull
+    @Size(min = 1, max = 255)
+    private String sureName;
+
+    @NotNull
+    @Email
+    @Size(min = 1, max = 255)
+    private String email;
+
+    @NotNull
+    @Size(min = 1, max = 255)
+    private String password;
+
+
+    public RegistrationUserDTO() {
+    }
+
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getSureName() {
+        return sureName;
+    }
+
+    public void setSureName(String sureName) {
+        this.sureName = sureName;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    @Override
+    public String toString() {
+        return String.format("User {name= %s, sureName= %s, email= %s, password= %s}",
+                name, sureName, email, password);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof RegistrationUserDTO)) {
+            return false;
+        }
+
+        final RegistrationUserDTO registrationUserDTO = (RegistrationUserDTO) o;
+
+        return (name != null ? name.equals(registrationUserDTO.name) : registrationUserDTO.name == null)
```

Comment:
reformat this code please. i guess that it would be more comfortable to read this part if this checks will be separated and only as final step will return something like "checkResult" variable or so...